src/Serilog.Sinks.MSSqlServer/LoggerConfigurationMSSqlServerExtensions.cs

```diff
@@ -45,14 +46,14 @@ public static class LoggerConfigurationMSSqlServerExtensions
             LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
             int batchPostingLimit = MSSqlServerSink.DefaultBatchPostingLimit,
             TimeSpan? period = null,
-            IFormatProvider formatProvider = null)
+            IFormatProvider formatProvider = null, bool utcTimestamp = true)
```

Comment:
Also, perhaps we could name it more specifically - something like `storeTimestampInUtc`? The current name at first glance sounds like it _is_ the timestamp. :-)